plugin/pkg/scheduler/api/types.go

```diff
@@ -118,6 +118,8 @@ type ExtenderConfig struct {
 	FilterVerb string
 	// Verb for the prioritize call, empty if not supported. This verb is appended to the URLPrefix when issuing the prioritize call to extender.
 	PrioritizeVerb string
+	// Verb for the bind call, empty if not supported. This verb is appended to the URLPrefix when issuing the bind call to extender.
+	BindVerb string
```

Comment:
The convention today is that this combines both the extender's intent to handle this method and the path where it is handled. If an extender didn't care about handling Bind OR if its an existing extender that doesn't handle binds at all, this is not populated. It helps preserve backward compatibility also.